chore: update DEVELOP.md (#341)
Orange-OpenSource/ouds-android: DEVELOP.md

## Documentation

OUDS Android documentation is generated using [Dokka](https://github.com/Kotlin/dokka).

You can test the documentation locally by launching the `dokkaGenerate` Gradle task.

```
./gradlew dokkaGenerate
```

> When generated, open `ouds-android/docs/dokka/index.html` in your browser.

Reviewer: Opening the `index.html` directly does not fully work because the lef side column is not visible. We should use the link provided by Dokka at the end of the task: http://localhost:63342/OUDS%20Android/docs/dokka/index.html. But I don't know if the port is always the same.

## Tests

Each component offered by OUDS Android has two kinds of tests: Android instrumented tests and [Paparazzi](https://github.com/cashapp/paparazzi) tests.

> The instrumented tests are in `ouds-android/core/src/androidTest` and allow to test interactions and behaviors.

Reviewer: I think that `ouds-android` should be removed of the path because it's not part of the GitHub repository.

The [Paparazzi](https://github.com/cashapp/paparazzi) tests are in `ouds-android/core/src/test`. They generate snapshots of the components in their various

> state (pressed, hover, focused, disabled, ...). These snapshots are compared on CI in order to identify possible regressions.  

Reviewer: ```suggestion
state (pressed, hover, focused, disabled, ...) and compare them to reference snapshots in order to identify possible regressions.  
```

> You can generate components snapshots by launching the `recordPaparazzi` or the `cleanRecordPaparazzi` Gradle task.

Reviewer: ```suggestion
You can generate reference snapshots of the components by launching the `recordPaparazzi` or the `cleanRecordPaparazzi` Gradle task.
```
